### PR TITLE
test: increase zmqlet test coverage (recv_message)

### PR DESCRIPTION
**Pull request by winstonww**
As per #692, this PR adds a unit test to test `recv_message` in the class `Zmqlet`.  The test verifies that `callback` passed into `recv_message` is called when message is sent and `msg_` that `callback` receives has the same data that the original `msg` has. Thank you.

### CODE REVIEW

File: tests/unit/test_is_read_message.py [hunk 1]
```diff
@@ -95,3 +96,24 @@ def test_not_read_zmqlet():
         d.tags['id'] = 2
         msg = Message(None, req, 'tmp', '')
         z.send_message(msg)
+
+
+def test_recv_message_zmqlet():
+    zmqlet1 = Zmqlet(args1, default_logger)
+    zmqlet2 = Zmqlet(args2, default_logger)
+    req = jina_pb2.RequestProto()
+    req.request_id = random_identity()
```
Comment (JoanFM) [inline tests/unit/test_is_read_message.py:105]: can u please use the request from the types module? we want to hide as much as possible Proto

File: tests/unit/test_is_read_message.py [hunk 2]
```diff
@@ -95,3 +96,24 @@ def test_not_read_zmqlet():
...
+    doc = req.index.docs.add()
+    doc.tags['id'] = 2
+    msg = Message(None, req, 'tmp', '')
+    called = False
+
+    def callback(msg_):
+        nonlocal called
+        called = True
```
Comment (JoanFM) [inline tests/unit/test_is_read_message.py:113]: use mocker as in many integration tests to ensure callback is called

**Review by JoanFM — CHANGES_REQUESTED** [review]
Thanks for your collaboration, just a small comment from my side

**Review by JoanFM — CHANGES_REQUESTED** [review]
watch out with the .swp

File: tests/unit/test_is_read_message.py (hunk 1 above)
Comment (winstonww) [reply tests/unit/test_is_read_message.py:106]: Thanks, addressed.

File: tests/unit/test_is_read_message.py (hunk 2 above)
Comment (winstonww) [reply tests/unit/test_is_read_message.py]: Thanks, addressed.

**Comment by winstonww** [comment]
Thank you for the comments.

**Comment by JoanFM** [comment]
recheckcla

**Comment by winstonww** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by JoanFM** [comment]
Hey @winstonww 

You may need to rebase on master to pass the black and docstring checks

**Review by JoanFM — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**